flocker_bb/builders/flocker_vagrant.py

```diff
@@ -357,6 +357,17 @@ def test_installed_package(box):
         tests='flocker',
         workdir='test',
     ))
+    # TODO: FLOC-253
+    # Quick ugly hack to get the test.log output out of the vagrant ssh
+    # trial run.
+    factory.addStep(ShellCommand(
+        name='dump-trial-test-log',
+        description=['dumping', 'trial', 'test', 'log'],
+        descriptionDone=['dump', 'trial', 'test', 'log', 'complete'],
+        command=['vagrant', 'ssh', '--', 'sudo', 'cat _trial_temp/test.log '],
```

Comment:
This command doesn't need `sudo`, since the files are created with read permissions for everyone. :question: